packages/react/src/components/Button/Button.js

```diff
@@ -53,7 +53,9 @@ const Button = React.forwardRef(function Button(
   const tooltipTimeout = useRef(null);
 
   const closeTooltips = (evt) => {
-    const tooltipNode = document?.querySelectorAll(`.${prefix}--tooltip--a11y`);
+    const tooltipNode = document?.querySelectorAll(
+      `.${prefix}--tooltip--a11y:not(.${prefix}--tooltip__trigger--definition)`
+    );
```

Comment:
I don't think this `:not` selector here is the ideal solution since you would also need to apply this to `TooltipIcon` in order to deal with the scenario where `TooltipIcon` and `TooltipDefinition` are on the same page

rather than adding this `:not` selector I think `TooltipDefinition` should have the same show/hide logic as `TooltipIcon` instead (unless there was a reason for it being excluded initially?)

Reply:
Makes sense 👍🏻

Reply:
@emyarod updated to toggle correctly. I didn't add the query selector as I feel like it's already a dirty pattern, and definition tooltips are not likely to be near other actionable icons where it would be getting in the way. Hovering over a button/tooltip icon will still close this tooltip, but hovering a tooltip definition will not close other tooltips.